src/marionette/html_element.cr

```diff
@@ -107,5 +107,14 @@ module Marionette
       options = options.merge(element: @id)
       @browser.save_screenshot(file, **options)
     end
+
```

Comment:
It would be nice to have some comment, what marionette is about to `select` and what should be provided as `value`.

Reply:
Added